docs/reference/security/authentication/configuring-file-realm.asciidoc

```diff
@@ -1,28 +1,33 @@
-You don't need to explicitly configure a `file` realm. The `file` and `native`
+The `file` and `native`
 realms are added to the realm chain by default. Unless configured otherwise, the
 `file` realm is added first, followed by the `native` realm.
 
+If you configure the realm chain manually,
+you must include the `file` realm for it to be used.
+
```

Comment:
This is not correct is it?
In 8.x the file realm is automatically included in the chain if it is not already present.

Reply:
🤦 I overlooked that the case that prompted me to change the docs was 7.17 cluster. And the 7.17 docs are correct. Thanks for catching this, and sorry for the noise.